samples/azure/webjob-host/sample.md

```diff
@@ -33,7 +32,7 @@ The `StartAsync` method of `IJobHost` is used to configure and start the endpoin
 
 snippet: WebJobHost_Start
 
-NOTE: If dependencies need to be shared between the service collection and NServiceBus infrastructure, such as message handlers, the [MSDependencyInjection NuGet package](/nservicebus/dependency-injection/msdependencyinjection.md) must be installed.
+NOTE: If dependencies need to be shared between the service collection and NServiceBus infrastructure, such as message handlers, see the [ASP.NET Core sample](/samples/dependency-injection/#related-samples).
```

Comment:
Given the link text says "ASP.NET Core sample", why are we not linking to that sample? I.e. `/samples/dependency-injection/aspnetcore/`.

Reply:
I couldn't get the docs engine to allow it since the sample is only available for v7.2.0 which is prerelease. Not sure how to resolve that so I'd say this is good enough for now

cc @DavidBoike